ecosystem/sep-direct-payment.md

```diff
@@ -0,0 +1,318 @@
+
+## Preamble
+
+```
+SEP: 00XX
+Title: Fiat-To-Fiat payments
+Author: SDF
+Status: Draft
+Created: 2020-04-07
+Updated: 2020-04-07
+Version 1.0.0
+```
+
+## Simple Summary
+
+This SEP defines a protocol for enabling fiat to fiat payments between two real world accounts, facilitated by two different anchors.
+
+## Abstract
+
+This proposal facilitates the ability for anchors to build a rail between two regions, allowing end users to send fiat from one bank account directly into another end users bank account.  In this flow, neither user needs to deal with the stellar network, the two anchors take care of everything for them.
+
+## Example
+
+Alice, in Nigeria, wants to send money to Bob in Europe. Alice signs up with NigeriaPay to make this payment to send money directly into Bob’s bank account. Bob doesn’t need to do anything, or know anything about this payment, besides letting Alice know what his bank account information is. Alice only needs to deal with her anchor (NigeriaPay).
+
+Cowrie will utilize its European rail, enabled with EuroPay Anchor service, to move the money to EuroPay in order to deposit it into Bob’s bank account.
+
+## Prerequisites
+
+* An anchor must define the location of their `DIRECT_PAYMENT_SERVER` in their [`stellar.toml`](sep-0001.md). Anchors will find each others servers by pulling this TOML file from their home domains.
+* Anchors will create bi-lateral agreements to interoperate with each other.  This differs from other protocols in that there is no concept of 'discoverability'.  Anchors should keep a mapping of their partnerships in different regions to their home domains.
+* Each anchor registers a stellar key with each counterparty anchor they interact with in order to identify themselves via [SEP-10 Web Authentication](sep-0010.md).
+
+
+## Authentication
+
+Anchors should support [SEP-10](sep-0010.md) web authentication to ensure the counterparty they're interoperating with is actually who they say they are.  Clients must submit the JWT obtained via the SEP-10 authentication flow to all API endpoints.
+
+The JWT should be included in all requests as request header:
+```
+Authorization: Bearer <JWT>
+```
+
+Any API request the fails to meet proper authentication should return a 403 forbidden response with the error payload:
+
+```
+{
+  "type": "authentication_required"
```

Comment:
Is there a reason we need this specific message? If we change the endpoints below to return 400 instead of 403 on a bad request, then a 403 can indicate bad authentication.